ui/components-react/src/test/TestUtils.ts

```diff
@@ -33,7 +33,8 @@ import { ITwinLocalization } from "@itwin/core-i18n";
 import { prettyDOM } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import type { AsyncValueProcessingResult } from "../components-react";
-import { DataControllerBase, UiComponents } from "../components-react";
+import { UiComponents } from "../components-react/UiComponents";
+import { DataControllerBase } from "../components-react/editors/PropertyEditorManager";
```

Comment:
Was this change done to resolve the no-internal-barrel-imports error or just code cleanup?

Reply:
This was for a lint error fix but a rule I updated should take care of this so I have reverted this change, good catch. Thanks

Reply:
Sounds good, thanks. Some of the auto fixes were breaking the build pipelines on core, so I used the --fix option only for the comma dangle error.

Reply:
Yup, I have also found that fixing no barrel imports breaks code coverage